compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp

```diff
@@ -576,13 +576,13 @@ struct ConvertCudnnBinaryOp : public CudnnOpConversionPattern<T> {
     MLIRContext *ctx = rewriter.getContext();
     ImplicitLocOpBuilder b(op->getLoc(), rewriter);
 
-    auto f32 = rewriter.getF32Type();
+    auto newType = rewriter.getF64Type();
```

Comment:
newType=>f64 (and in few other places as well)

Reply:
Fixed.